src/contexts/Server.context.tsx

```diff
@@ -66,40 +66,40 @@ export function ServerProvider({ children }: ServerProviderProps): JSX.Element {
   const reloadProperties = useCallback(async () => {
     setIsReloading(true);
     if (instance) {
+      const tasks: [() => Promise<unknown>, string][] = [
+        [async () => setHostname(await instance.system.getHostname()), 'hostname'],
+        [async () => setIdentConfig(await instance.unraid.getIdentConfig()), 'ident'],
+        [async () => setSystemInfo(await instance.system.info()), 'info'],
+        [async () => setCaseModel(await instance.unraid.getCaseModel()), 'case'],
+        [async () => setDiskUsage(await instance.system.diskfree()), 'diskfree'],
+        [async () => setNotifications(await instance.unraid.getNotifications()), 'notifications'],
+        [
+          async () => {
+            const { raw } = await instance.system.uptime();
+            if (DEBUG) {
+              log.debug('got uptime');
+            }
+            const parsedUptime = parse(raw, 'yyyy-MM-dd HH:mm:ss', new Date());
+            setUptime(parsedUptime);
+          },
+          'uptime',
+        ],
+      ];
```

Comment:
Maybe later we want to put this in a separate function so it's a bit easier to read but for now go for it